episodes/what-is-an-image.md

```diff
@@ -0,0 +1,540 @@
+icon on the left side.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+Let's look at the human mitosis image more closely - copy the following into 
+Napari's console:
+
+```python
+
+# Get the image data for the first layer in Napari
+image = viewer.layers[0].data
+
+# Print the image values and type
+print(image)
+print(type(image))
+```
+
+```output
+[[ 8  8  8 ... 63 78 75]
+ [ 8  8  7 ... 67 71 71]
+ [ 9  8  8 ... 53 64 66]
+ ...
+ [ 8  9  8 ... 17 24 59]
+ [ 8  8  8 ... 17 22 55]
+ [ 8  8  8 ... 16 18 38]]
+ 
+ <class 'numpy.ndarray'>
+```
+
+You should see that a series of numbers are printed out that are stored in a 
+`numpy.ndarray`. Fundamentally, this means that all images are really just 
+arrays of numbers (one number per pixel). Napari is reading those values and 
+converting them into squares of particular colours for us to see in the viewer, 
+but this is only to help us interpret the image contents - the numbers are the 
+real underlying data.
+
+For example, look at the simplified image of an arrow below. On the left is the 
+array of numbers, with the corresponding image display on the right:
+
+![](fig/array.png){alt="A diagram comparing the array of numbers and image 
+display for a simplified image of an arrow"}
+
+In Napari this array is a `numpy.ndarray`. [NumPy](https://numpy.org/) is a 
+popular python package that provides 'n-dimensional arrays' (or 'ndarray' for 
+short). N-dimensional just means they can support any number of dimensions - 
+for example, 2D, 3D and beyond (like time series, images with many 
+channels etc.).
+
+## Creating an image
+
+Where do the numbers in our image array come from? The exact details of how an 
+image is created will depend on the type of microscope you are using e.g. 
+widefield, confocal, superresolution etc. In general though, we have 3 main 
+parts: 
+
+- **Sample:** the object we want to image e.g. some cells
+- **Objective lens:** the lens that gathers the light and focuses it for 
+detection
+- **Detector:** the device that detects the light to form the digital image 
+e.g. a CCD camera
+
+To briefly summarise, light is emitted from the sample (e.g. from a fluorescent 
+marker), then gathered and focused with the objective lens, before hitting the 
+detector. The detector is divided into many small, light sensitive areas - these 
+are physical pixels, that will correspond to the pixels in the final image. When 
+light hits one of these areas it is converted to one or more electrons, with 
+more light resulting in more electrons and a higher final value for that pixel.
+
+The important factor to understand is that the final pixel value is only ever an 
+approximation of the real sample. Many factors will affect this final result 
+including the microscope optics, detector performance etc.
+
+::::::::::::::::::::::::::::::::::::: discussion 
+
+Read the ['A simple microscope' section of Pete Bankhead's bioimage book](
```

Comment:
This book looks a really good resource. It makes me wonder how best to use this. Is the idea to make it as recommended reading ahead of the course? Or is the idea that we will stop for a few minutes while they read this section and then have a discussion?

Reply:
I was more thinking the latter - stopping for a few minutes to read this section + discuss. I think it's a bit large to have as recommended reading before the course, but I'll definitely link to it at the end as further reading - as it really is an excellent resource! I've started an issue to collect useful links/resources for the end: https://github.com/HealthBioscienceIDEAS/microscopy-novice/issues/29